README.md

```diff
@@ -1,4 +1,4 @@
-> Note: PMM2 is out! see https://github.com/percona/pmm/tree/PMM-2.0  
+> Note: The `master` branch points to PMM v1, which is not being actively developed. PMM v2 offers much more functionality, so please check out https://github.com/percona/pmm/tree/PMM-2.0  
```

Comment:
Let's use the term "default" instead of keywords we want to remove.

Reply:
But there is no `default` branch (it could be a valid branch name). This will create a lot of confusion. And we can't change that branch name to something else for technical reasons.